src/writers.ts

```diff
@@ -1,5 +1,13 @@
 import { evm } from '@snapshot-labs/checkpoint';
 import { Payment } from '../.checkpoint/models';
+import { readFileSync } from 'fs';
+import { getJSON } from './utils';
+
+const tokens = JSON.parse(readFileSync('./src/payment_tokens.json', 'utf8'));
```

Comment:
You don't need `fs` to load a JSON.
```suggestion
import { getJSON } from './utils';
import tokens from './payment_tokens.json';
```

Reply:
JS magic